packages/react-silk/src/components/Button.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react'
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+} from 'react-native'
+
+const styles = StyleSheet.create({
+  container: {
+    alignSelf: 'flex-start',
+  },
+  button: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: 'row',
+    paddingVertical: 10,
+    paddingHorizontal: 15,
+  },
+  primaryButton: {
```

Comment:
I think you should define the color palette in `constants` for reuse in others place